styleguide/source/_patterns/00-atoms/02-text/02-h2/_h2.scss

```diff
@@ -8,13 +8,15 @@
 body {
   h2, .h2, %h2 {
     @include type($font-serif, 1.222em, $font-weight-medium, 1.1818181818);
+    margin-bottom: 6px;
```

Comment:
@martensc I think so? I can't 100% remember, but I was basing this on how other spacing was done on other components. I think because the spacing is so specific, defaulting the px values is what we decided. But I'd love to ask @deadarm if she remembers specifically.